packages/fast-components-styles-msft/src/outline-button/index.ts

```diff
@@ -3,13 +3,7 @@ import { ComponentStyles } from "@microsoft/fast-jss-manager";
 import { applyFocusVisible, format, subtract, toPx } from "@microsoft/fast-jss-utilities";
 import { DesignSystem } from "../design-system";
 import { baseButton, buttonStyles } from "../patterns/button";
-import {
-    neutralFocus,
-    neutralForegroundRest,
-    neutralOutlineActive,
-    neutralOutlineHover,
-    neutralOutlineRest,
-} from "../utilities/color";
+import { neutralFocus, neutralForegroundRest, neutralOutline } from "../utilities/color";
```

Comment:
I think the outline button example given is a good one - I'd like to see how the swapping gets implemented here. Would it be possible to create a `stories.tsx` file with one example similar to what you detailed in the write-up? 

> an alternate implementation, like outlines are neutral colored by default, and go to accent on hover or active states.

Reply:
I've tried this, and I found that this code is not working quite as expected. I'm having a problem that seems to be something related to webpack, where the function value I'm putting into the design system is getting swapped out with some 'reviver' function that doesn't seem to be resolving correctly.

I suspect we can do something like this with some sort of design system registry, where we could potentially even look up or pass in named resources from the standard or custom design system. I don't understand enough about webpack to resolve this though.